src/js/common/components/questions/PrivacyAgreement.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ErrorableCheckbox from '../form-elements/ErrorableCheckbox.jsx';
+
+export default function PrivacyAgreement({ onChange, checked }) {
+  return (
+    <div>
+      <ErrorableCheckbox required
+          checked={checked}
+          onValueChange={onChange}
+          label={<span>I have read and accept the <a href="/privacy-policy">privacy policy</a></span>}/>
```

Comment:
Getting some propType warnings for the `label` on `ErrorableCheckbox` because it's expecting a string. Not a huge deal but maybe something we can fix later.